modules/codemod/lib/v5/spec/codemod-v5.spec.ts

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+import {defineInlineTest} from 'jscodeshift/dist/testUtils';
+import transform from '..';
+
+describe('Canvas Kit v5 Codemod', () => {
+  defineInlineTest(
+    transform,
+    {},
+    `
+  import canvas from '@workday/canvas-kit-react';
+      `,
+    `
+  import canvas from "@workday/canvas-kit-react/core";
+      `,
+    'Changes default canvas bundle import to core import'
+  );
+
+  defineInlineTest(
+    transform,
+    {},
+    `
+  import * as foo from '@workday/canvas-kit-react-button';
+      `,
+    `
+  import * as foo from "@workday/canvas-kit-react/button";
+      `,
+    'Updates namespaced import'
+  );
+
+  // TODO: ADD HANDLING FOR @workday/canvas-kit-react source. Need to split imports
+  // defineInlineTest(
+  //   transform,
+  //   {},
+  //   `
+  // import * as foo from '@workday/canvas-kit-react';
+  //     `,
+  //   `
+  // import * as foo from "@workday/canvas-kit-react/core";
+  //     `,
+  //   'Changes namespaced bundle import to namespaced core import'
+  // );
```

Comment:
This will be a headache to account for as we need to convert this into separate imports from all of the mapped modules, then assign them all to an object called `foo`. Thoughts on whether it's worth handling this case? I can't imagine it's too common.

Reply:
I suppose if someone files an issue, we could try updating based on their usage.